### ISSUE TITLE
Can symbols, besides referencing files, also include the function of referencing # titles?

### ISSUE THREAD

**Issue by Wooden-Gear** [logancyang/obsidian-copilot#747]
[[]] 符号除了引用文件之外，能不能加入引用# 标题的功能

Can symbols, besides referencing files, also include the function of referencing # titles?

**Comment by logancyang** [comment]
What does `# titles` mean? Do you mean tags? You can reference tags or folders with custom prompt templating https://www.obsidiancopilot.com/en/docs/

**Comment by Wooden-Gear** [comment]
> What does `# titles` mean? Do you mean tags? You can reference tags or folders with custom prompt templating https://www.obsidiancopilot.com/en/docs/代码 id=0 的`# titles`意味着什么？你是想说标签吗？你可以通过自定义提示模板引用标签或文件夹，参考https://www.obsidiancopilot.com/en/docs/

不是标签。
是以下这个语法：
[[文件名#标题名称]]
只引用某个文件中的某个标题下的内容
而不是引用整个文件

Not a tag.  
The following syntax: 
[[filename#headingname]] 
only references content under a specific heading in a specific file.
Instead of referencing the entire file

**Comment by logancyang** [comment]
Oh I see. It is quite a nice addition if implemented.

**Comment by varg1714** [comment]
> Oh I see. It is quite a nice addition if implemented.

I'm following this issue with great interest. I completely understand that priorities can shift, but I was curious to know if this feature is being considered for a future release. Knowing this would help me plan my own work around it.

Thanks again for creating and maintaining this project! 😋

**Comment by ichts** [comment]
> > Oh I see. It is quite a nice addition if implemented.
> 
> I'm following this issue with great interest. I completely understand that priorities can shift, but I was curious to know if this feature is being considered for a future release. Knowing this would help me plan my own work around it.
> 
> Thanks again for creating and maintaining this project! 😋

Yes. It's on our roadmap.

**Comment by specialadapter** [comment]
I too would love to reference headings as I follow the "Map of content" model of personal knowledge management and I link lots of related notes under specific headings on a topic-specific note.

e.g.

## Artificial Intelligence

[[Tokens]]

[[Parameters]]

[[Interference]]

[[Context]]

etc.

 Asking Copilot to questions about notes linked under a specific section would be absolutely amazing.

P.S. I hardly used #Tags anymore and keep my folders to an absolute minimum. 

Thanks in advance.

**Comment by ZB333ZB** [comment]
I'm really looking forward to this feature. Could references to text blocks like [[filename#^block_ID]] be implemented as well? Thank you

**Comment by logancyang** [comment]
Thanks for reporting this, and sorry for the slow reply.

Copilot v3 is in code freeze, so we're closing this as not planned. v4 is a ground-up rewrite and is launching soon, with a very different architecture around agent mode.

If this is still an issue for you in v4 once it lands, please open a new issue and we'll take a fresh look. 🙏
